CHANGELOG.md

```diff
@@ -79,6 +80,84 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.13.0
+
+### Change
+
+- refactor(ai-proxy): move read_response into ai_driver.request function [#12101](https://github.com/apache/apisix/pull/12101)
+- refactor: mcp server framework implementation #[12168](https://github.com/apache/apisix/pull/12168)
+- change: mark server-info plugin as deprecated [#12244](https://github.com/apache/apisix/pull/12244)
```

Comment:
Also not a breaking change as the plugin is not yet removed. Maybe you can put it under plugins section.

In the release notes on the website, I will make a deprecations section for this.